src/logic/screenTransactionHooks.js

```diff
@@ -149,6 +149,27 @@ type SignTransactionArgs = {
   parentAccount: ?Account,
 };
 
+export const broadcastSignedTx = async (
+  account: AccountLike,
+  parentAccount: ?Account,
+  signedOperation: SignedOperation,
+): Promise<Operation> => {
+  invariant(account, "account not present");
+  const mainAccount = getMainAccount(account, parentAccount);
+  const bridge = getAccountBridge(account, parentAccount);
+
+  if (getEnv("DISABLE_TRANSACTION_BROADCAST")) {
+    return Promise.resolve(signedOperation.operation);
+  }
+
+  return execAndWaitAtLeast(3000, () =>
+    bridge.broadcast({
+      account: mainAccount,
+      signedOperation,
+    }),
+  );
+};
```

Comment:
I see this is exactly the content of useBroadcast so I think let's have useBroadcast use this code 🤔 
Also I would happy accept a PR to add this in live-common =)

Reply:
Removed code duplication, moving to LLC will be done in a next step.